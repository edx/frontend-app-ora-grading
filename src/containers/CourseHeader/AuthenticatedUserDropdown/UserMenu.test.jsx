import React from 'react';
import { shallow } from 'enzyme';

import { UserMenu } from './UserMenu';

jest.mock('@edx/frontend-platform', () => ({
  getConfig: () => ({
    LMS_BASE_URL: '<LMS_BASE_URL>',
    LOGOUT_URL: '<LOGOUT_URL>',
    SUPPORT_URL: '<SUPPORT_URL>',
  }),
}));

describe('Header AuthenticatedUserDropdown UserMenu component', () => {
  const props = {
    intl: { formatMessage: (msg) => msg.defaultMessage },
    username: 'test-username',
  };
  test('snapshot', () => {
    expect(
      shallow(<UserMenu {...props} />),
    ).toMatchSnapshot();
  });
});
