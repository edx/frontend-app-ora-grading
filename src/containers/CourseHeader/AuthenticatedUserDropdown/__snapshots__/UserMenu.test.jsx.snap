// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header AuthenticatedUserDropdown UserMenu component snapshot 1`] = `
<Dropdown.Menu
  className="dropdown-menu-right"
>
  <Dropdown.Item
    href="<LMS_BASE_URL>/dashboard"
  >
    Dashboard
  </Dropdown.Item>
  <Dropdown.Item
    href="<LMS_BASE_URL>/u/test-username"
  >
    Profile
  </Dropdown.Item>
  <Dropdown.Item
    href="<LMS_BASE_URL>/account/settings"
  >
    Account
  </Dropdown.Item>
  <Dropdown.Item
    href="<LOGOUT_URL>"
  >
    Sign Out
  </Dropdown.Item>
</Dropdown.Menu>
`;
